nservicebus/sqlserver/operations-scripting.md

```diff
@@ -31,6 +31,10 @@ snippet:sqlserver-powershell-nativesend
 
 snippet:sqlserver-nativesend-usage
 
+The `MessageTypeToSend` dictionary value represents the message .NET type, by default the endpoint receiving the message will use that message type to deserialize the incoming message. Depending on the [versioning](/samples/versioning/sample.md) strategy the type can be expressed in 2 different ways:
```

Comment:
@indualagarsamy I think here would make even more sense http://docs.particular.net/nservicebus/messaging/headers. The message owner doco needs update too (snippets?), but this particular piece of info would fit headers better imo.